lib/dns.js

```diff
@@ -31,7 +72,7 @@ Client.prototype.query = function query(type, host, cb) {
     ra: true
   });
 
-  if (host[host.length - 1] !== '.')
+  if (host[ host.length - 1 ] !== '.')
```

Comment:
Yea, actually, I just tried cloning bn.js and ran jscs with the .jscsrc there, and it has lots of complaints. Maybe your local setup uses a different .jscsrc, like one from your home dir.